frontends/precinct-scanner/src/screens/poll_worker_screen.tsx

```diff
@@ -179,7 +179,8 @@ export function PollWorkerScreen({
       setCurrentTally(tally);
     }
     void calculateTally();
-  }, [election, scannedBallotCount, precinctSelection]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I saw in tests how this effect may run multiple times and reload CVRs, when in reality we only want to run it when the poll worker screen loads. The CVRs from the backend cannot update while the poll worker screen is open (no ballots can be cast).

Reply:
Did the previous dependencies change and trigger the effect again? If it does get triggered again, does that cause bugs?